pkg/validation/attributes/test-fixtures/commands/apply.yaml

```diff
@@ -0,0 +1,3 @@
+# Attributes are defined in test-fixtures/attributes/attributes-referenced.yaml
+label: "{{version}}"
+component: "{{foo}}"
```

Comment:
Trailing newlines missing in various files here, may as well fix it now in case we get a linter one day.